cmd/podman/pod_ps.go

```diff
@@ -431,21 +435,23 @@ func getPodTemplateOutput(psParams []podPsJSONParams, opts podPsOptions) ([]podP
 			infraID = shortID(infraID)
 		}
 		for _, ctrInfo := range psParam.CtrsInfo {
-			ctrStr += "[ "
+			infoSlice := make([]string, 0)
```

Comment:
I'm assuming you meant to lose the brackets?  "[ string ]".

Reply:
I actually opted to keep the brackets, I just removed to space between the brackets and the info. This fix was mostly because the output was [ <ctrID><ctrName> <ctrStatus> ] when they should be space separated. I also tried to do comma separated without the brackets but I think it looks worse.